getting-started/quickstart-guides/php.md

```diff
@@ -42,425 +42,1483 @@ It will prompt a few options. Select defaults by pressing enter.
+| >                             | gt                      |  
+| >=                            | gte                     |  
+| <                             | lt                      |  
+| <=                            | lte                     |  
+
+
+## Server-side File Upload
+
+The SDK provides a simple interface using the `$imageKit->upload()` or `$imageKit->uploadFile()` method to upload files to the [ImageKit Media Library](https://imagekit.io/dashboard/media-library). 
+
+- [Check all the supported file types and extensions](https://docs.imagekit.io/api-reference/upload-file-api#allowed-file-types-for-uploading).
+- [Check all the supported parameters and details](https://docs.imagekit.io/api-reference/upload-file-api/server-side-file-upload).
+
+#### Example
+```php
+$uploadFile = $imageKit->upload([
+    'file' => 'your_file',              //  required, "binary","base64" or "file url"
+    'fileName' => 'your_file_name.jpg', //  required
+]);
+```
+#### Response
+```json
+{
+    "error": null,
+    "result": {
+        "fileId": "6286329dfef1b033aee60211",
+        "name": "your_file_name_S-PgGysnR.jpg",
+        "size": 94466,
+        "versionInfo": {
+            "id": "6286329dfef1b033aee60211",
+            "name": "Version 1"
+        },
+        "filePath": "/your_file_name_S-PgGysnR.jpg",
+        "url": "https://ik.imagekit.io/demo/your_file_name_S-PgGysnR.jpg",
+        "fileType": "image",
+        "height": 640,
+        "width": 960,
+        "thumbnailUrl": "https://ik.imagekit.io/demo/tr:n-ik_ml_thumbnail/your_file_name_S-PgGysnR.jpg",
+        "tags": [],
+        "AITags": null,
+        "customMetadata": { },
+        "extensionStatus": {}
+    },
+    "responseMetadata":{
+        "headers":{
+            "access-control-allow-origin": "*",
+            "x-ik-requestid": "e98f2464-2a86-4934-a5ab-9a226df012c9",
+            "content-type": "application/json; charset=utf-8",
+            "content-length": "434",
+            "etag": "W/"1b2-reNzjRCFNt45rEyD7yFY/dk+Ghg"",
+            "date": "Thu, 16 Jun 2022 14:22:01 GMT",
+            "x-request-id": "e98f2464-2a86-4934-a5ab-9a226df012c9"
+        },
+        "raw":{
+            "fileId": "6286329dfef1b033aee60211",
+            "name": "your_file_name_S-PgGysnR.jpg",
+            "size": 94466,
+            "versionInfo": {
+                "id": "6286329dfef1b033aee60211",
+                "name": "Version 1"
+            },
+            "filePath": "/your_file_name_S-PgGysnR.jpg",
+            "url": "https://ik.imagekit.io/demo/your_file_name_S-PgGysnR.jpg",
+            "fileType": "image",
+            "height": 640,
+            "width": 960,
+            "thumbnailUrl": "https://ik.imagekit.io/demo/tr:n-ik_ml_thumbnail/your_file_name_S-PgGysnR.jpg",
+            "tags": [],
+            "AITags": null,
+            "customMetadata": { },
+            "extensionStatus": {}
+        },
+        "statusCode":200
+    }
+}
 ```
+#### Optional Parameters
```

Comment:
We should link docs rather than specifying a list here which might change in the future.